src/components/logo/Logo.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from '../../styles/logo/Logo.scss';
+
+export const Logo = () => {
+	return (
+		<div className={styles.stage}>
+			<div className={styles.layer}></div>
```

Comment:
we could use  the `.map` method here to repeat similar items

Reply:
The code has been refactored with the `.map` method.